rpms/generate_rpm_package.sh

```diff
@@ -173,8 +173,12 @@ main() {
             ;;
         "-a"|"--architecture")
             if [ -n "$2" ]; then
-                ARCHITECTURE="$2"
-                shift 2
+                if [[ "$2" == "x86_64" ]] || [[ "$2" == "i386" ]]; then
```

Comment:
This check must consider amd64 also

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/7227907bae4d24ba9920560ff50f6c40e0a7dc8f